docs/source/gbd2017_models/index.rst

```diff
@@ -8,5 +8,5 @@ GBD 2017 Models
    :maxdepth: 2
    :glob:
 
-   *
    */index
+   concept_models/vivarium_conceptmodel_template/concept_model
```

Comment:
Oh, I guess that answers my question above about whether the page shows up in the index.